configs/katello/3.9.yaml

```diff
@@ -73,137 +66,28 @@
     based_off: katello-thirdparty-pulp-rhel7
     arches:
       - x86_64
-  - name: katello-3.9-thirdparty-pulp-rhel6
-    based_off: katello-thirdparty-pulp-rhel6
-    arches:
-      - x86_64
-  - name: katello-3.9-thirdparty-pulp-rhel5
-    based_off: katello-thirdparty-pulp-rhel5
-    arches:
-      - x86_64
   - name: katello-3.9-rhel7
     based_off: katello-nightly-rhel7
     helper_tags:
       katello-3.9-thirdparty-rhel7: katello-thirdparty-rhel7
       katello-3.9-rhel7-override: null
-      katello-3.9-client-rhel7: null
     build_target: katello-3.9-rhel7-build
     build_package_group_source_tag: katello-nightly-rhel7-build
     arches:
       - x86_64
     inherits:
       katello-3.9-rhel7-build:
         katello-3.9-rhel7-override: 0
+        tfm-ror52-rhel7: 2
```

Comment:
what does the 2 indicate here?

Reply:
This indicates the priority relative to other tags.